README.md

```diff
@@ -63,6 +63,12 @@ git clone https://github.com/microsoft/WinUI-Gallery.git
 
 Ensure that the `WinUIGallery` project is set as the startup project in Visual Studio.
 
+> [!WARNING]
+> Try deleting [`nuget.config`](nuget.config) and building again if you get the following build error (https://github.com/microsoft/WinUI-Gallery/issues/1659):

```

Comment:
Let's add a readable text to the link, maybe even mote it out of the brackets?

Reply:
Thank you. I rewrote it.